lib/utils/request.js

```diff
@@ -42,8 +42,13 @@ function createErrorMessage(err, httpResponse, body){
     err.requestId = requestId;
     return err;
   } else {
-    var msg = util.format(i18n._('Error - not 2xx status code. (%s)\n'), statusCode);
-
+    var errorMsgTxt = '';
+    if(statusCode === 401 || statusCode === 403){
```

Comment:
Welcome in RHMAP :) It depends if we calling supercore or millicore, but I agree to leave it this way.